packages/explorable-explanations/src/protectedAudience/modules/bubbles.js

```diff
@@ -278,6 +286,7 @@ bubbles.reverseBarrageAnimation = async (index) => {
   }
 
   document.getElementById('interest-canvas').style.zIndex = 4;
+  document.getElementById('bubble-container-div').style.zIndex = 2;
```

Comment:
Thinking why we are querying the DOM twice `'bubble-container-div'`.

Reply:
If zIndex is more than 3 when barrage animation is running then the bubbles appear to be going behind the container rather than on the container.